src/core/particles/webgl/ParticleRenderer.js

```diff
@@ -210,7 +205,7 @@ ParticleRenderer.prototype.render = function ( container )
             return;
         }
 
-        if(!this.properties[0].dynamic || !this.properties[3].dynamic)
+        if(!container._properties[0] || !container._properties[3])
```

Comment:
This is where the bug would have occurred as the status of the properties of the renderer was checked rather than the status of the properties of the container.
The renderer being shared by several containers the status might not be corresponding to the current container and lead to the static data not being uploaded.